Buildings/Controls/OBC/ASHRAE/PrimarySystem/ChillerPlant/Staging/Processes/Subsequences/ResetMinBypass.mo

```diff
@@ -0,0 +1,208 @@
+﻿within Buildings.Controls.OBC.ASHRAE.PrimarySystem.ChillerPlant.Staging.Processes.Subsequences;
+block ResetMinBypass
+  "Sequence for reset minimum chilled water flow setpoint"
+
+  parameter Modelica.SIunits.Time aftByPasSetTim = 60
+    "Time after setpoint achieved";
+  parameter Real relFloDif=0.05
+    "Relative error to the setpoint for checking if it has achieved flow rate setpoint"
+    annotation (Dialog(tab="Advanced"));
+
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uUpsDevSta
+    "Status of resetting status of device before reset minimum flow setpoint"
+    annotation (Placement(transformation(extent={{-200,60},{-160,100}}),
+      iconTransformation(extent={{-140,60},{-100,100}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanInput uStaCha
+    "Indicate if there is stage change"
+    annotation (Placement(transformation(extent={{-200,20},{-160,60}}),
+      iconTransformation(extent={{-140,20},{-100,60}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput VChiWat_flow(
+    final min=0,
+    final unit="m3/s",
+    final quantity="VolumeFlowRate") "Measured chilled water flow rate"
+    annotation (Placement(transformation(extent={{-200,-40},{-160,0}}),
+      iconTransformation(extent={{-140,-60},{-100,-20}})));
+  Buildings.Controls.OBC.CDL.Interfaces.RealInput VMinChiWat_setpoint(
+    final min=0,
+    final unit="m3/s",
+    final quantity="VolumeFlowRate") "Minimum chiller water flow setpoint"
+    annotation (Placement(transformation(extent={{-200,-100},{-160,-60}}),
+      iconTransformation(extent={{-140,-100},{-100,-60}})));
+  Buildings.Controls.OBC.CDL.Interfaces.BooleanOutput yMinBypRes
```

Comment:
What is this an input to?

Reply:
Check if the setpoint has been reset successfully and it will be flag to indicate that it is time to act next staging step.